Review Response Render/templates/main.pug

```diff
@@ -0,0 +1,12 @@
+html(lang="en")
+	head
+		meta(charset="UTF-8")
+		meta(http-equiv="X-UA-Compatible", content="IE=edge")
+		meta(name="viewport", content="width=device-width, initial-scale=1.0")
+		title Document
+	body 
+		section
+			ul
+				each post in posts
+					li= post.body
+					
```

Comment:
@ct-aa, let's change the indention from tabs to spaces and set it to 2 spaces. Also, please add a new line at the EOF.

Reply:
Thanks, @lycoangelo. This is now updated.